src/files/searchFile.api.js

```diff
@@ -49,11 +63,6 @@ function parseFilters(obj) {
     result.fileName = fileName;
   }
 
-  if (Reflect.has(obj, "fileExtension")) {
-    let fileExtension = Reflect.get(obj, "fileExtension");
-    result.fileExtension = fileExtension;
-  }
-
   if (
```

Comment:
filter method structure as an example. 
```ts
type Filer = {
  filename: string,
  createdAt: {
    from: Date,
    to: Date
  },
  isShared?: boolean
}
```

need to use `POST` http verb for this.